app/controllers/embedded_terraform_repository_controller.rb

```diff
@@ -0,0 +1,163 @@
+class EmbeddedTerraformRepositoryController < ApplicationController
+  before_action :check_privileges
+  before_action :get_session_data
+
+  after_action :cleanup_action
+  after_action :set_session_data
+
+  include Mixins::GenericListMixin
+  include Mixins::GenericSessionMixin
+  include Mixins::GenericShowMixin
+  include Mixins::BreadcrumbsMixin
+
+  menu_section :embedded_terraform_repository
+
+  def self.display_methods
+    %w[configuration_script_payloads]
+  end
+
+  def self.custom_display_modes
+    %w[output]
+  end
+
+  def self.model
+    ManageIQ::Providers::EmbeddedTerraform::AutomationManager::ConfigurationScriptSource
+  end
+
+  def show_searchbar?
+    true
+  end
+
+  def title
+    _("Repository")
+  end
+
+  def button
+    case params[:pressed]
+    when "embedded_configuration_script_source_edit"
+      id = params[:miq_grid_checks]
+      javascript_redirect(:action => 'edit', :id => id)
+    when "embedded_configuration_script_source_add"
+      javascript_redirect(:action => 'new')
+    when "embedded_terraform_repositories_reload"
+      show_list
+      render :update do |page|
+        page << javascript_prologue
+        page.replace("gtl_div", :partial => "layouts/gtl")
+      end
+    when "embedded_terraform_repository_reload"
+      show
+      render :update do |page|
+        page << javascript_prologue
+        page.replace("main_div", :template => "embedded_terraform_repository/show")
+      end
+    when "ansible_repository_tag"
+      tag(self.class.model)
+    when "embedded_configuration_script_payload_tag" # workflows from nested list
+      tag(ManageIQ::Providers::EmbeddedTerraform::AutomationManager::ConfigurationScriptPayload)
```

Comment:
From: https://github.com/ManageIQ/manageiq-providers-embedded_terraform/pull/9
```suggestion
      tag(ManageIQ::Providers::EmbeddedTerraform::AutomationManager::Template)
```